File: LeetCode/Challenge/Angle Between Hands of a Clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double h = hour * 30 + (double)minutes / 2;
        double m = 6 * minutes;
        
        double result = abs(h-m);
        
        return result > 180 ? 360 - result : result;
    }
};

File: LeetCode/Challenge/Angle Between Hands of a Clock.go
func angleClock(hour int, minutes int) float64 {
    var h float64 = float64(hour) * float64(30) + float64(minutes) / float64(2);
    var m float64= 6 * float64(minutes);

    var result float64 = h-m;
    
    if result < 0{
        result = -result;
    }

    if result > 180{
        return 360 - result;
    }else{
        return result;
    }
}